tests/unit/commands/local/invoke/test_cli.py

```diff
@@ -29,10 +31,15 @@ def setUp(self):
 
     @patch("samcli.commands.local.invoke.cli.InvokeContext")
     @patch("samcli.commands.local.invoke.cli._get_event")
-    def test_cli_must_setup_context_and_invoke(self, get_event_mock, InvokeContextMock):
+    @patch("samcli.commands.local.invoke.cli._get_debug_context")
+    def test_cli_must_setup_context_and_invoke(self, get_debug_context_mock, get_event_mock, InvokeContextMock):
         event_data = "data"
         get_event_mock.return_value = event_data
 
+        debug_context_data = DebugContext(debug_port=self.debug_port, debug_args=self.debug_args,
+                                          debugger_path=self.debugger_path)
+        get_debug_context_mock.return_value = debug_context_data
```

Comment:
This needs to change because `get_debug_context` method won't exist

Reply:
will update the test where needed.